types/rrdtool/tslint.json

```diff
@@ -0,0 +1,14 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "npm-naming": [
+            true,
+            {
+                "mode":"code",
+                "errors": [
+                    ["NoDefaultExport", false]
```

Comment:
without the `export default`, you shouldn't need to disable this rule either.

Reply:
amended